.circleci/config.yml

```diff
@@ -160,6 +160,16 @@ commands:
           paths:
             - 'dist'
 jobs:
+  check-for-merge-conflicts:
+    executor: telegraf-ci
+    steps:
+      - checkout
+      - run: |
+          git config --global user.email "telegraf@influxdb.com"
+          git config --global user.name "telegraf"
+          git remote add upstream https://github.com/influxdata/telegraf
+          git fetch upstream
```

Comment:
@sspaink any update? I think it will remove a lot of logs in the CI results (especially because now only fetching master)

Reply:
Oops I forgot about this, thanks for the reminder. Your suggestion seems good to me, I do need to rework this so that the rebase only for pull requests and not for release.

Reply:
Okay, I also see this happening in the current CI checks, but I can't find out where this is configured so I can make a PR to change/update them. Can you point me to it?

Reply:
I removed the automatic rebase on master from CI: https://github.com/influxdata/telegraf/pull/11293 If it is still happening then that PR should rebase on master to get the latest CI were the rebase step is removed.